.pre-commit-hooks.yaml

```diff
@@ -78,3 +78,12 @@
   require_serial: true
   types: [jupyter]
   additional_dependencies: [pyupgrade]
+- id: nbqa-yapf
+  name: nbqa-yapf
+  description: "Run 'yapf' on a Jupyter Notebook"
+  entry: nbqa yapf --in-place
```

Comment:
I'm not sure how the `--in-place` modification flag interacts with `--nbqa-mutate` but this seemed to behave like I wanted.